src/launchWizard/launchWizard.ts

```diff
@@ -325,13 +294,22 @@ class LaunchWizard {
     let stopOnEntry = defaultValues.stopOnEntry ? 'checked' : ''
     let trace = defaultValues.trace ? 'checked' : ''
     let useExistingServer = defaultValues.useExistingServer ? 'checked' : ''
-    let daffodilDebugClasspathAction =
-      defaultValues.daffodilDebugClasspathAction === 'append'
-        ? defaultConf.daffodilDebugClasspath
-        : 'replace'
-    let replaceCheck =
-      daffodilDebugClasspathAction !== 'append' ? 'checked' : ''
-    let appendCheck = daffodilDebugClasspathAction === 'append' ? 'checked' : ''
+
+    let daffodilDebugClasspathList =
+      '<ul id="daffodilDebugClasspathTable" style="list-style: none; padding-left: 20px;">'
+    if (defaultValues.daffodilDebugClasspath) {
+      let itemArray = defaultValues.daffodilDebugClasspath.split(':')
+      for (var i = 0; i < itemArray.length; i++) {
+        daffodilDebugClasspathList += `
+          <li style="margin-left: -5px;" onclick="removeDebugClasspathItem(this)">
+            <p id="debug-classpath-li-${itemArray[i]}" class="debug-classpath-item">
+              <button id="remove-debug-classpath-li-${itemArray[i]}" class="minus-button" type="button">-</button>
+              ${itemArray[i]}
+            </p>
+          </li>`
```

Comment:
This feels much like the code in the `updateClasspathList` function.  Is there any way to consolidate it?

Reply:
Not really. This is done for the initial setup of the list in the extension code if there are classpath items that need appended at the start, where as `updateClasspathList` requires using JS to get the list and append items to it. This can not be done inside of the extensions TS code as it does have have access to the DOM. The JS code could possibly on startup/load initialize the list, however doing it this way could cause issues rendering, such as the UI loading once then immediately getting refreshed causing the render to not seem smooth or possibly breaking. It would also cause the need to embed the classpath items into a hidden html element. There is some wiring that can be done between the JS and TS that could possibly do what you are saying, however even if it could in this cause it wouldn't be effective and would also probably cause more issues/confusion than it would fix.

Reply:
The Data Editor feature will be doing just as you described, sending data from the extension to the webview for rendering.  We'll see how it goes.  Either way we'll get smarter on this pattern.

Reply:
Yeah so as I said this is not to bad to do but in this situation I don't think it makes to much sense, as I don't think it provides any benefits over the way it is currently being done. I believe changing it to try to have JS add items on load, can cause issues during the rendering making the launch config wizard not open smoothly or at all.